src/tests/frontend/cypress/integration/collaborate.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+Cypress.Commands.add('iframe', {prevSubject: 'element'},
+    ($iframe) => new Cypress.Promise((resolve) => {
+      $iframe.ready(() => {
+        resolve($iframe.contents().find('body'));
+      });
+    }));
+
+describe(__filename, () => {
+  it('Pad content exists', () => {
```

Comment:
All 4 browsers are all doing the same thing, yes? I think it would be better if this collaboration test tested collaboration:

1. browser 1 types "hello world 1" while the other three wait for that to appear
2. browser 2 types "hello world 2" while the other three wait for that to appear
3. etc.